lib/request.js

```diff
@@ -107,6 +108,8 @@ function Request (options) {
     this.headers['content-length'] = (Buffer.isBuffer(payload) ? payload.length : Buffer.byteLength(payload)).toString()
   }
 
+  Object.keys(this.headers).forEach(header => this.rawHeaders.push(header, this.headers[header]))
```

Comment:
Could you please use a for...of loop instead?

Reply:
Sure, I will

Reply:
done